src/main/resources/ApplicationInsights.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ApplicationInsights xmlns="http://schemas.microsoft.com/ApplicationInsights/2013/Settings" schemaVersion="2014-05-30">
     <!-- The key from the portal: -->
-    <InstrumentationKey>d3278c59-22d7-4697-af8c-ff0077e3b4e9</InstrumentationKey>
+    <InstrumentationKey>xxxxxx-xxxx-xxxx-xxxx-xxxxxxxxx-xxxx</InstrumentationKey>
```

Comment:
Can we use any placeholder here?

Reply:
your mean like `${your-instrument-key}` ? I use `xxxxx` as it looks intuitive.